tests/test_list.py

```
@@ -958,52 +964,54 @@ def check(actual, expect):
-    monkeypatch.setattr(MetadataFactory, "fetch_http", lambda self, _: update_xml)
+def test_show_list_tools_long_qtdesignstudio(capsys, monkeypatch, columns, expect):
+    _html = (Path(__file__).parent / "data" / "mac-desktop-tools_qtdesignstudio.html").read_text("utf-8")
+    update_xml = (Path(__file__).parent / "data" / "mac-desktop-tools_qtdesignstudio-update.xml").read_text("utf-8")
+
+    def _mock_fetch_http(self, rest_of_url, *args, **kwargs):
+        if rest_of_url.endswith("/"):
+            return _html
+        if rest_of_url.endswith("/Updates.xml"):
+            return update_xml
+        else:
+            assert False, f"Fetched an unexpected file at '{rest_of_url}'"
```

Ruff (0.15.7)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)